### ISSUE TITLE
Error on macOS Sonoma

### ISSUE THREAD

**Issue by udoschmitz** [JoshuaWalsh/aegisub-timewarp#2]
I use this version of Aegisub on my Mac:

https://github.com/arch1t3cht/Aegisub/releases/tag/feature_11

Timewarp throws error:

```
Lua reported a runtime error:
    File "/Applications/Aegisub.app/Contents/SharedSupport/automation/autoload/timewarp.lua", line 69
<anonymous function at lines 54-70>
    File "<C function>", line -1
?
Subtitles object is no longer valid
```

.srt-file is unchanged.

**Comment by joenroll** [comment]
Same Issue here

**Comment by YoshiWalsh** [comment]
I think this might be an upstream issue. The line  in this automation that produces the error is a simple call to `set_undo_point`. The actual error occurs within Aegisub's C code.

**Comment by udoschmitz** [comment]
jfyi: I commented out line 69 `-- aegisub.set_undo_point(script_name)`
no errors now, script changes start and end times but just not how it was supposed to. Am I missing something?

**Comment by YoshiWalsh** [comment]
Please provide details of which settings you are giving the script (or a screenshot).

**Comment by joshtch** [comment]
Not sure if this is the same issue, but I was getting this error too before I realized that you have to enter the times as decimal numbers like `1483.76`, not time codes such as `24:43.76` (Aegisub doesn't allow typing colon characters, but will silently accept them when pasted). Aegisub/Lua/Timewarp should have spit out an error well before getting to the C code, but did not.

**Comment by udoschmitz** [comment]
Yes, with decimal values I don’t get the error. Can Aegisub give me decimals or do I have to use the calculator?

**Comment by chocmake** [comment]
> Yes, with decimal values I don’t get the error. Can Aegisub give me decimals or do I have to use the calculator?

Might like to check out [a fork](https://github.com/chocmake/aegisub-timewarp) that supports regular timestamp syntax.
